### ISSUE TITLE
The music keeps fading in/out during replays

### ISSUE THREAD

**Issue by Arthandas** [Damgam/Assetto-Corsa-Dynamic-Music-Player#1]
I disabled all fading out options but the music keeps fading out during replays. I'm using v2.0.4.

**Comment by Damgam** [comment]
Can you specify a bit more what exactly are you doing during these replays and which settings you have enabled/disabled, so I can reproduce the problem on my end? Make sure you're running the required version of CSP.

**Comment by Arthandas** [comment]
I put all my music in the "other" folder. I have all fade out options disabled. It works fine during a race, but when I finish it and play the replay (I use the "cinematic" auto camera) it behaves like the fadeout options are enabled, so depending on the camera angle and nearby cars the song sometimes gets quieter. I'll test it more later, I temporarily uninstalled AC. Btw, I'm using CSP v0.2.4

**Comment by Damgam** [comment]
Ok. That should be enough details. Will see what I can do. I have bugfixing patch planned anyway as there are a few bugs i've already found.

**Comment by Arthandas** [comment]
One more thing, would it be possible to disable music "triggers" be removing the folders so the program doesn't even search it? For example, putting everything it "other" and removing "idle" would not play any music during idling. I'm asking for this feature because I find it slightly annoying, that when I start the race a song starts playing during the setup phase (where you can tune the car) and when I'm done with it the program ends the song and starts a different one. If you don't tune your car (or just quickly load a preset) this phase is very quick so you get to hear only a few seconds of a song before another song is loaded after a proper race start.

**Comment by Damgam** [comment]
I'm pretty sure that Idle is already optional. If it's not, consider that a bug. You can also just put some silent audio file in idle if you want pitstop to have no music at all.

**Comment by Damgam** [comment]
https://github.com/Damgam/Assetto-Corsa-Dynamic-Music-Player/releases/tag/ReleaseV2.1

**Comment by Arthandas** [comment]
Sorry, but the music still sometimes fade outs during replays, also "idle" music trigger is not optional, I still get music before race start and a new track after starting (I didn't try the silent file yet).

**Comment by Damgam** [comment]
Oh right because theres that race countdown behaviour... OK i'll make that optional for the next update.

**Comment by Arthandas** [comment]
Hey, is seems the music fadeout is not the program's fault. I disabled DMP and played the music in foobar in the background and the music was still fading out where there were other cars nearby. Apparently it's caused by Windows.

**Comment by Damgam** [comment]
Ohhhh you're audio clipping. Yea that makes much sense.

I recommend turning down Master volume of Assetto Corsa down. I run mine at 30%

**Comment by Arthandas** [comment]
But I like the current volume :) Any other workarounds?

**Comment by Damgam** [comment]
Nope. You're quite literally reaching the volume limit of digital audio, and at that point audio driver either turns down the volume, or gives you really ugly clipping... Yours is turning down volume.

**Comment by Arthandas** [comment]
I don't understand, it doesn't happen in any other games.

**Comment by Damgam** [comment]
Yea because other games are often either better at managing the volumes of stuff, or their 100% isn't so high. With Assetto you also have the aspect of many modded cars being much louder than normal, that also causes problems.
